lec04-dplyr.Rmd

```diff
@@ -1,4 +1,755 @@
 ---
-title: "Introduction to data wrangling using the dplyr R package"
+title: Data wrangling and visualization in the tidyverse
 ---
 
+
+> ### Learning Objectives
+>
+> - Understand the split-apply-combine concept for data analysis.
+> - Use `summarize`, `group_by`, and `tally` to split a data frame into groups
+> of observations, apply a summary statistics for each group, and then combine
+> the results.
+> - Produce scatter plots, line plots, and histograms using ggplot.
+> - Set universal plot settings.
+> - Understand and apply faceting in ggplot.
+> - Explore scientific questions using dplyr and ggplot.
+
+
+> ### Lesson outline
+>
+> - Split-apply-combine techniques in **`dplyr`** (15min)
+> - Using `tally` to summarize categorical data (10 min)
+> - Plotting with **`ggplot2`** (10 min)
+> - Building plots iteratively (10 min)
+> - Split-apply-combine... plot! (10 min)
+> - Faceting (10 min)
```

Comment:
Is it intentionally less than the lecture time? ~1 hr 5 min?

Reply:
Yeah, I wanted to be more conservative. But clearly it's better to just add ~5min to everything and have the total time be more accurate. I will do that and then merge.